test/linters/accessibility/aria_label_is_well_formatted_test.rb

```diff
@@ -20,25 +20,30 @@ def test_warns_when_aria_label_starts_with_downcase
     assert_equal 5, @linter.offenses.count
   end
 
+  def test_warns_when_aria_label_contains_newline_characer
+    @file = <<~HTML
+      <button aria-label="Go to my&#10;website." ></button>
+      <button aria-label="Go to my\nwebsite." ></button>
+    HTML
+    @linter.run(processed_source)
+    assert_equal 2, @linter.offenses.count
+  end
+
   def test_does_not_warn_when_aria_labelledby_starts_with_downcase
     @file = "<button aria-labelledby='some-element-1'</button>"
     @linter.run(processed_source)
 
     assert_empty @linter.offenses
   end
 
-  def test_does_not_warn_when_aria_label_starts_with_anything_other_than_downcase
-    @file = <<~HTML
-     <button aria-label="Submit" ></button>
-     <button aria-label="Close" ></button>
-     <button aria-label="11 open" ></button>
-    HTML
+  def test_does_not_warn_with_string_interpolation
+    @file = '<button aria-label="Add a cat <%= "or dog " if dog_allowed? %>pet"></button>'
```

Comment:
This `aria-label` value wasn't being parsed correctly in `possible_attribute_values`.
 
`possible_attribute_values(tag, "aria-label")` was returning `or dog`, which is causing it to be flagged. This appears to because `basic_conditional_code_check` is trying to parse the conditional, but it's not working very well. This `basic_conditional_code_check` was introduced long before we pulled this into this library, and I never examined closely what it was doing. It doesn't seem to be doing what it's supposed to do so I'm removing it.